docker/docker-compose.deployment.yml

```diff
@@ -0,0 +1,166 @@
+      BLOCKCHAIN_URL: ${BLOCKCHAIN_URL:?BLOCKCHAIN_URL-cannot-be-blank}
+      CIRCOM_WORKER_HOST: ${CIRCOM_WORKER_HOST:-worker}
+      DEPLOY_MOCK_TOKENS: ${DEPLOY_MOCK_TOKENS?DEPLOY_MOCK_TOKENS-not-set}
+      DEPLOY_MOCKED_SANCTIONS_CONTRACT: ${DEPLOY_MOCKED_SANCTIONS_CONTRACT:?DEPLOY_MOCKED_SANCTIONS_CONTRACT-cannot-be-blank}
+      ETH_ADDRESS: ${ETH_ADDRESS:?ETH_ADDRESS-cannot-be-blank}
+      ETH_PRIVATE_KEY: ${ETH_PRIVATE_KEY:?ETH_PRIVATE_KEY-cannot-be-blank}
+      ETH_NETWORK: ${ETH_NETWORK:?ETH_NETWORK-cannot-be-blank}
+      FEE_L2_TOKEN_ID: ${FEE_L2_TOKEN_ID:?FEE_L2_TOKEN_ID-cannot-be-blank}
+      GAS: ${GAS}
+      GAS_PRICE: ${GAS_PRICE}
+      MULTISIG_APPROVERS: ${MULTISIG_APPROVERS?MULTISIG_APPROVERS-not-set}
+      MULTISIG_SIGNATURE_THRESHOLD: ${MULTISIG_SIGNATURE_THRESHOLD:?MULTISIG_SIGNATURE_THRESHOLD-cannot-be-blank}
+      SKIP_DEPLOYMENT: ${SKIP_DEPLOYMENT:-'false'}
+      WHITELISTING: ${WHITELISTING?WHITELISTING-not-set}
+      UPGRADE: ${UPGRADE_CONTRACTS}
+
+  mongodb:
+    image: mongo:4.4.1-bionic
+    hostname: mongodb
+    ports:
+      - 27017:27017
+    networks:
+      - nightfall_network
+    command: --quiet
+    volumes:
+      - type: volume
+        source: mongodb
+        target: /data/db
+
+  rabbitmq:
+    image: rabbitmq
+    ports:
+      - '15674:15674'
+      - '5672:5672'
+    networks:
+      - nightfall_network
+
+  optimist:
+    image: ghcr.io/eyblockchain/nightfall3-optimist:latest
+    build:
+      dockerfile: docker/optimist.Dockerfile
+      context: ..
+    depends_on:
+      - mongodb
+    networks:
+      - nightfall_network
+    ports:
+      - 8081:80
+      - 8082:8080
+    volumes:
+      - ./volumes/build:/app/build/
+    environment:
+      AUTOSTART_RETRIES: ${AUTOSTART_RETRIES:-600}
+      BLOCKCHAIN_URL: ${BLOCKCHAIN_URL:?BLOCKCHAIN_URL-cannot-be-blank}
+      ENVIRONMENT: ${ENVIRONMENT:?ENVIRONMENT-cannot-be-blank}
+      HASH_TYPE: poseidon
+      IS_CHALLENGER: 'false'
+      LOG_LEVEL: ${LOG_LEVEL:-debug}
+      LOG_HTTP_PAYLOAD_ENABLED: ${LOG_HTTP_PAYLOAD_ENABLED:-true}
+      LOG_HTTP_FULL_DATA: ${LOG_HTTP_FULL_DATA:-false}
+      MAX_BLOCK_SIZE: ${MAX_BLOCK_SIZE:-50000}
+      MONGO_URL: ${OPTIMIST_MONGO_URL:-mongodb://mongodb:27017}
+      PROPOSER_MAX_BLOCK_PERIOD_MILIS: ${PROPOSER_MAX_BLOCK_PERIOD_MILIS:-0}
+      USE_EXTERNAL_NODE: ${USE_EXTERNAL_NODE}
+      WEBSOCKET_PORT: 8080
+    command: ['npm', 'run', 'start']
+
+  worker:
+    image: ghcr.io/eyblockchain/nightfall3-worker:latest
+    build:
+      dockerfile: docker/worker.Dockerfile
+      context: ..
+    volumes:
+      - ./volumes/proving_files:/app/output/
+    depends_on:
+      - rabbitmq
+    networks:
+      - nightfall_network
+    environment:
+      ENVIRONMENT: ${ENVIRONMENT:-local}
+      LOG_LEVEL: ${LOG_LEVEL:-debug}
+      LOG_HTTP_PAYLOAD_ENABLED: ${LOG_HTTP_PAYLOAD_ENABLED:-true}
+      LOG_HTTP_FULL_DATA: ${LOG_HTTP_FULL_DATA:-false}
+    entrypoint: ['npm', 'run', 'start']
```

Comment:
?

```suggestion
    command: ['npm', 'run', 'start']
```

Reply:
FYI There's another instance like this one later on in `docker/docker-compose.yml`

Reply:
Yes, but the deployment doesn't rely on any other docker-compose, it should be able to do the work without any other dependency. I decided to do this to simplify things.